scripts/build.js

```diff
@@ -29,8 +29,8 @@ const dist = "dist";
     }
 
     for (const f of await fs.readdir(src)) {
-      if (/\.(?:js|css|html)$/.test(f)) {
-        await fs.copyFile(path.join(src, f), path.join(dist, f));
+      if (/\.(?:js|css|html)$/.test(f) || f === "assets") {
+        await fs.cp(path.join(src, f), path.join(dist, f), {recursive: true});
```

Comment:
I am getting this warning 

"(node:11802) UnhandledPromiseRejectionWarning: TypeError: fs.cp is not a function
    at /Users/Ash/Desktop/GameDev/cs195-project/scripts/build.js:33:18
(Use `node --trace-warnings ...` to show where the warning was created)
(node:11802) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:11802) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code."